src/components/TimerModal/TimerBundle.js

```diff
@@ -0,0 +1,32 @@
+import {useContext, useEffect} from "react";
+import {PlayContext, TimerContext} from "./TimerContextProvider";
+import Timer from "./Timer";
+import TimerPlay from "./TimerPlay";
+import style from "./Modal.module.scss";
+
+export default function TimerBundle({timerTitle}) {
+
+    const [timer, setTimer] = useContext(TimerContext)
+    const [play, setPlay] = useContext(PlayContext)
+
+    useEffect(() => {
+
+        if (play) {
+            setTimeout(() => setTimer(timer.seconds !== 0 && {seconds: timer.seconds - 1}), 1000);
+        }
```

Comment:
Would we need to clearTimeout when the timer is paused?

Reply:
Thanks for that